standards/naming-domains.md

```diff
@@ -4,25 +4,33 @@ category: Operating services
 # Naming domains
 
 All services with real users and which look like GOV.UK should be on
```

Comment:
real users? What do we mean by that term?

Reply:
I meant people using it for real to get something done - so not prototypes or services at an early stage of development which aren't yet ready for users to have access. That could mean private beta onwards, but not everything we do goes through those phases in the same way.

There's probably a better way of describing that than "with real users".

Reply:
“All citizen facing services which look like GOV.UK”?

Reply:
This standard really needs to cover internal services, since there's not much in the service manual about them. I was intending this section to cover both internal and public-facing services - do you think we should be doing something different for internal things?

Reply:
I've removed all references to “real users” and extended it to “all MOJ services” and then tried to clear up what that means. Hopefully that fixes it?